config/manager/manager.yaml

```diff
@@ -85,9 +85,6 @@ spec:
           initialDelaySeconds: 5
           periodSeconds: 10
         resources:
-          limits:
-            cpu: 500m
-            memory: 128Mi
           requests:
             cpu: 10m
             memory: 64Mi
```

Comment:
Could we bump this to 128 ?... 

Would that make sense ? How much do we use in Kubemacpool ?

Reply:
[20Mi](https://github.com/k8snetworkplumbingwg/kubemacpool/blob/main/config/release/kubemacpool.yaml#L325C13-L325C25)

Reply:
opps, my bad
[100Mi](https://github.com/k8snetworkplumbingwg/kubemacpool/blob/main/config/release/kubemacpool.yaml#L325C13-L325C25)

Reply:
ok, sold ... Let's stick to this number, and implement paging in a follow-up. I'll create a ticket.

Reply:
I think 128Mi is not too low, not too high.. why not.

Reply:
Added..